apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chat/ComposeView.kt

```diff
@@ -180,7 +180,7 @@ fun ComposeView(
         isImage(uri) -> {
           // Image
           val drawable = getDrawableFromUri(uri)
-          bitmap = if (drawable != null) getBitmapFromUri(uri) else null
+          bitmap = getBitmapFromUri(uri)
```

Comment:
Why condition is removed? This code below is very hard to follow...

Reply:
`getDrawableFromUri` needed on Android because it can say is it AnimatedDrawable or not. There is no such thing on Desktop and this function returns null always. So if on desktop it returns null and the check `if (drawable != null)` is here, bitmap will be null too on desktop which is not what I want. 

`if (drawable != null)` was needed just to make less work on Android: if `getDrawableFromUri` returns null there, no file actually exists so no need to try to get bitmap from it. But as I said, this optimization will not work with desktop,

Reply:
can we make this optimisation conditional on the platform then?

Reply:
Let CPU do it's 1 ms work